sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -7,9 +7,13 @@
 ## 12.9.0 (2020-11-11)
 - Fixed a bug where interspersed element types returned by page listing would deserialize incorrectly.
 - Fixed a bug where BlobInputStream would not eTag lock on the blob, resulting in undesirable behavior if the blob was modified in the middle of reading. 
+- Renamed BlobDownloadToFileOptions.rangeGetContentMd5 to BlobDownloadToFileOptions.retrieveContentRangeMd5.
+- Added support for the 2020-04-08 service version. 
+- Added support to upload block blob from URL.
```

Comment:
These should be under a new version right?

Reply:
Yea. Good catch.